src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -98,8 +102,12 @@ export class EditDataGridPanel extends GridParentComponent {
 		@ILogService protected logService: ILogService
 	) {
 		super(contextMenuService, keybindingService, contextKeyService, configurationService, clipboardService, queryEditorService, logService);
-		this.nativeElement = document.createElement('editdatagridpanel');
-		this.nativeElement.className = 'slickgridContainer';
+		this.nativeElement = document.createElement('div');
+		this.nativeElement.className = 'editDataGridPanel';
+		this.nativeElement.classList.add('slickgridContainer');
+		// Assign measurements to prevent override
+		this.nativeElement.style.height = this.tableHeight;
```

Comment:
Just put these in the CSS file

Reply:
I did so, but they were overridden when the container was attached to the panel

Reply:
Nevermind, it turns out I entered string values rather than numerical values, it works now.